bin/iabc.js

```diff
@@ -0,0 +1,21 @@
+#!/usr/bin/env node
+import make from '../src/cli/make.js';
+import create from '../src/cli/curl-route-builder.js';
+
+// Retrieve command-line arguments
+const args = process.argv.slice(2);
+const command = args[0];
+
+if (command === undefined) {
+    make(process.argv, process.cwd(),);
+}
+else {
+    if (command === "curl" || command === "har" || command === "open-api"){
+        create(process.argv, process.cwd(),);
```

Comment:
shouldn't this call different creates? process.argv needs to be manipulated somewhere to fit the old implementation - or the old implementation must change